pkg/providers/amifamily/bottlerocket.go

```diff
@@ -38,17 +38,42 @@ type Bottlerocket struct {
 	*Options
 }
 
-// SSMAlias returns the AMI Alias to query SSM
-func (b Bottlerocket) SSMAlias(version string, instanceType *cloudprovider.InstanceType) string {
-	arch := "x86_64"
-	amiSuffix := ""
-	if !resources.IsZero(instanceType.Capacity[v1alpha1.ResourceNVIDIAGPU]) {
-		amiSuffix = "-nvidia"
-	}
-	if instanceType.Requirements.Get(v1.LabelArchStable).Has(v1alpha5.ArchitectureArm64) {
-		arch = v1alpha5.ArchitectureArm64
+// DefaultAMIs returns the AMI name, and Requirements, with an SSM query
+func (b Bottlerocket) DefaultAMIs(version string) []DefaultAMIOutput {
+	return []DefaultAMIOutput{
+		{
+			Name:  fmt.Sprintf("bottlerocket-aws-k8s-%s%s", version, "-x86_64"),
```

Comment:
```suggestion
			Name:  fmt.Sprintf("bottlerocket-aws-k8s-%s-x86_64", version),
```

If you have a static string in the `fmt.Sprintf`, why not just put that static string in the templated string? I think this applies to all the `fmt.Sprintf`s so check this across all the files